packages/fx-core/src/question/create.ts

```diff
@@ -1357,22 +1357,51 @@ export function officeAddinHostingQuestion(): SingleSelectQuestion {
   };
 }
 
-export function OfficeAddinFrameworkQuestion(): SingleSelectQuestion {
-  return {
+export function officeAddinFrameworkQuestion(): SingleSelectQuestion {
+  const officeAddinFrameworkQuestion: SingleSelectQuestion = {
```

Comment:
Just wonder that why just follow the original style to return it directly to keep the change as samll as possible?

Reply:
Just only as consistent as possible with other functions.